qemux86-64.robot

```diff
@@ -14,12 +15,24 @@ Preparing test environment
   File Should Exist     ${image}  msg="Provided images file does not exist"
   Set Suite Variable    ${application_repo}    https://github.com/resin-io-projects/alpine-barebone.git
   Resin login with email %{email} and password %{password}
+  Set Suite Variable    ${mount_destination}    /mnt
+  Set Suite Variable    ${host_os_partition}    2
+  Set Suite Variable    ${path_to_fingerprint}  ${mount_destination}/resin-root.fingerprint
 Adding new SSH key
   Add new SSH key with name ${application_name}
 Deleting application if it already exists
   Force delete application ${application_name}
 Creating application
   Create application ${application_name} with device type ${device_type}
+Check host OS fingerprint file
+  Set up loop device for "${image}"
+  Find the last loop device that was assigned
+  Set Suite Variable    ${path_to_mount}    /dev2/loop${LOOPDEVICE}p${host_os_partition} ${mount_destination}
+  Set Suite Variable    ${path_to_detach}   /dev2/loop${LOOPDEVICE}
+  Mount "${path_to_mount}" file system using loop device
+  Verify resin-root.fingerprint in "${path_to_fingerprint}"
+  Unmount "${mount_destination}" file system using the loop device
```

Comment:
Since unmount and mount are technically not dependent on the loop device - could we remove the loop device term from the keywords?